eod_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  Arm_R1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_R2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_R3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_R4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R4_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_R5_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R5_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_R6_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_R6_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  Gripper_finger1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joints: gripper_finger1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  Arm_L1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_L2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_L3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_L4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L4_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_L5_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L5_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  Arm_L6_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: Arm_L6_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  

 